cloud/google/clientcomputeservice.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package google
+
+import (
+	compute "google.golang.org/api/compute/v1"
+)
+
+type GCEClientComputeService interface {
```

Comment:
@spew I moved this into it's own file in the google package since it's used by both machineactuator and clusteractuator. What do you think of this pattern?

Reply:
That seems like the right thing to do to me.

Reply:
Great, thanks! Can you read through this one more time and add the lgtm label if everything looks good?

Reply:
Will do -- I was just reading through this and noticed that log nit that i noted above.

Reply:
oops I missed that one! Should be fixed now.